common/src/main/java/com/netflix/conductor/common/metadata/tasks/Task.java

```diff
@@ -124,6 +124,8 @@ public boolean isRetriable(){
 	private WorkflowTask workflowTask;
 	
 	private String domain;
+
+	private Boolean started;
```

Comment:
Using object boolean so we can easily identify which legacy tasks and which are new (since this commit)